common-tasks/usb.md

```diff
@@ -21,199 +21,108 @@ redirect_from:
-
-
-Removing a USB qube
--------------------
-
-**Warning:** This procedure will result in your USB controller(s) being attached
-directly to dom0.
-
-1. Shut down the USB qube.
-2. In Qubes Manager, right-click on the USB qube and select "Remove VM."
-3. Open the file `/etc/default/grub` in dom0.
-4. Find the line(s) that begins with `GRUB_CMDLINE_LINUX`.
-5. If `rd.qubes.hide_all_usb` appears anywhere in those lines, remove it.
-6. Save and close the file.
-7. Run the command `grub2-mkconfig -o /boot/grub2/grub.cfg` in dom0.
-8. Reboot.
-
-
-Security Warning about USB Input Devices
-----------------------------------------
-
-**Important security warning. Please read this section carefully!**
-
-If you connect USB input devices (keyboard and mouse) to a VM, that VM will effectively have control over your system.
-Because of this, the benefits of using a USB qube are much smaller than using a fully untrusted USB qube.
-In addition to having control over your system, such VM can also sniff all the input you enter there (for example, passwords in the case of a USB keyboard).
-
-There is no simple way to protect against sniffing, but you can make it harder to exploit control over input devices.
-
-If you have only a USB mouse connected to a USB qube, but the keyboard is connected directly to dom0 (using a PS/2 connector, for example), you simply need to lock the screen when you are away from your computer.
-You must do this every time you leave your computer unattended, even if there no risk of anyone else having direct physical access to your computer.
-This is because you are guarding the system not only against anyone with local access, but also against possible actions from a potentially compromised USB qube.
+How to attach USB drives
+------------------------
 
-If your keyboard is also connected to a USB qube, things are much harder.
-Locking the screen (with a traditional password) does not solve the problem, because the USB qube can simply sniff this password and later easily unlock the screen.
-One possibility is to set up the screen locker to require an additional step to unlock (i.e., two-factor authentication).
-One way to achieve this is to use a [YubiKey], or some other hardware token, or even to manually enter a one-time password.
+(**Note:** In the present context, the term "USB drive" denotes any
+[USB mass storage device][mass-storage]. In addition to smaller flash memory
+sticks, this includes things like USB external hard drives.)
 
-How to use a USB keyboard
--------------------------
+Qubes OS supports the ability to attach a USB drive (or just one or more of its
+partitions) to any qube easily, no matter which qube actually handles the USB
+controller. (The USB controller may be assigned on the **Devices** tab of a
+qube's settings page in Qubes VM Manager or by using the
+[qvm-pci][Assigning Devices] command. For guidance on finding the correct USB
+controller, see [here][usb-controller].)
```

Comment:
This suggests that connecting USB controller to a VM is a common action, and may even encourage users to do it each time USB drive is connected. I'd clearly separate "creating USB qube" section (one time setup) and "using USB qube" (common actions).

Reply:
I was trying to clearly separate them with https://github.com/QubesOS/qubes-doc/pull/539. How else would you suggest?